server/src/api/api.graphql

```diff
@@ -78,3 +84,54 @@ type Category {
     # The human-readable, user-facing name for this category
     category_name: String!
 }
+
+enum RequestStatus {
+	SUBMITTED
+	APPROVED
+	DENIED
+	ABANDONED
+    CANCELLED
+	READY_FOR_PICKUP
+	FULFILLED
+	RETURNED
+	LOST
+	DAMAGED
+}
+
+type Request {
+	request_id: Int!,
+	user: User!,
+	item: Item!,
+	status: RequestStatus!,
+	quantity: Int!,
+	createdAt: String!,
+	updatedAt: String!
+}
+
+# Simple representation of a request without the linked item or user
+type SimpleRequest {
```

Comment:
Can you elaborate on this?  I don't really know how to do this in GraphQL types

Reply:
Try this?
https://stackoverflow.com/questions/48917863/how-do-you-extend-types-in-graphql

Reply:
This doesn't exactly do what we want.  You can extend types but it's not inheritance perse, so you can't, for instance, create Request off of SimpleRequest.  Since it's such a small number of properties (and not likely to grow) I'm fine with leaving it, or I could make the item and user fields optional (remove the `!`) on Request and then just use one object.